flocker/restapi/_infrastructure.py

```diff
@@ -58,16 +74,10 @@ def _logging(original):
     """
     @wraps(original)
     def logger(self, request, **routeArguments):
-        try:
-            logger = self.logger
-        except AttributeError:
-            logger = _logger
-        else:
-            if logger is None:
-                logger = _logger
+        logger = _get_logger(self)
 
-        path = repr(request.path).decode("ascii")
-        action = REQUEST(logger, request_path=path)
+        action = REQUEST(logger, request_path=request.path,
+                         method=request.method)
```

Comment:
Did Eliot ever gain the ability to serialize a single field into multiple outputs?  `request=request` would be more convenient here.

I guess even if Eliot serializers aren't allowed to do that, there could still be an `IRequest` field serializer that spits out something like `{"path": ..., "method", ...}`.  That makes it a lot easier to improve the logging without fixing all of the (oh so very, very many...) call sites of `REQUEST`.